tests/functional/test_completion.py

```diff
@@ -176,7 +176,7 @@ def test_completion_alone(autocomplete_script: PipTestEnvironment) -> None:
     assert (
         "ERROR: You must pass --bash or --fish or --powershell or --zsh"
         in result.stderr
-    ), ("completion alone failed -- " + result.stderr)
+    ), "completion alone failed -- " + result.stderr
```

Comment:
Hmm, I prefer black's approach of leaving the parentheses alone here. But I guess the whole point of using an automated formatter is that I'm supposed to not have opinions like that 🙄